plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -52,9 +52,19 @@ class MantisMarkdown extends Parsedown
 	private $table_class = null;
 
 	/**
-	 * @var string inline style
+	 * @var int plugin configuration
 	 */
-	private $inline_style = null;
+	private $config_process_buglinks = OFF;
+
+	/**
+	 * @var int plugin configuration
+	 */
+	private $config_process_urls = OFF;
```

Comment:
The properties are initialized in constructor, so it's pointless to assign a default value.

The same goes for *$table_class*, null is the default anyway.